frontend/src/pages/pipelines/global/modelCustomization/landingPage/NextStepsSection.tsx

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
-import { useNavigate } from 'react-router';
-import { Accordion, Button, Content, ContentVariants, Label } from '@patternfly/react-core';
+import { Link } from 'react-router';
```

Comment:
Just one more note, we might want to always import from `react-router-dom` instead of `react-router` for all the web developments referring to https://github.com/remix-run/react-router/issues/4648.

Reply:
Ah, checking https://stackoverflow.com/a/79254507. Since we are using v7, looks like it doesn't matter anymore. I think we can go ahead with it. 😄